lib/rules/max-nesting-depth/README.md

```diff
@@ -148,6 +148,125 @@ a {
 }
 ```
 
+### `ignore: ["pseudo-classes"]`
+
+Ignore pseudoclases.
```

Comment:
Typo: `pseudo-classes`.

Reply:
In my first approach the first two examples were allowed but I believe what you say may make sense. I'll think about it a bit more and I'll try to propose a solution for this case.

The main reason for doing this rule was to prevent declarations like...
```
.a {
  .b:hover {}
}
```
and allow 
```
.a {
  .b {
    &:hover {}
  }
}
```

Thanks!

Reply:
This is how it should work in my opinion

__1st example__
I agree with you because `b` selector is not `:hover` dependent and it's a kind of trick to use it like that. I need to fix this.

__2nd example__
I think it's the main purpose of this issue, I give you a real example for this use case
http://jsfiddle.net/inspiratweb/r1d6qpne/ in my opinion it works as it should because it's just a selector for that nesting level.

__3rd example__
It's OK because although they are different selectors, both of them are pseudo-classes.
It works like...
```
a {
  &:hover {
     top: 0;
  }
  &:focus {
    top: 0;
  }
}
```
Do you agree with me?

Reply:
Sounds good to me.

Thanks for working it through!